util/tls/tls_socket.h

```diff
@@ -7,12 +7,12 @@
 #include <openssl/ssl.h>
 
 #include "util/fiber_socket_base.h"
-
+#include "util/tls/tls_engine.h"
 
 namespace util {
 namespace tls {
 
-class Engine;
+// class Engine;
```

Comment:
?

Reply:
I had to include the file with the Engine class because of the "Handshake" function that I've added here

Reply:
i understood, so why not delete the line?

Reply:
Also, see my comment below. I do not think handshake must be exposed.

Reply:
removed